search/search-client/src/main/java/io/camunda/search/clients/impl/NoopSearchClientsProxy.java

```diff
@@ -5,7 +5,7 @@
  * Licensed under the Camunda License 1.0. You may not use this file
  * except in compliance with the Camunda License 1.0.
  */
-package io.camunda.zeebe.engine.search;
+package io.camunda.search.clients.impl;
```

Comment:
🔧 We could consider adding a static factory to `SearchClientsProxy` to ensure no-one (not the zeebe/engine module, nor ZPT) has to interact with the impl classes directly.

For example:

```java
public interface SearchClientsProxy {

  /* .. */

  static SearchClientsProxy noop() {
    return new NoopSearchClientsProxy();
  }
}
```

Reply:
Happy to unblock this by committing and pushing the change. Let me know @megglos

Reply:
Also happy to just take this over as medic

Reply:
Sure would appreciate if you can take this from here add the method and merge! Will you also adapt ZPT to make use of it?

Reply:
Yes, will do!